appinfo/info.xml

```diff
@@ -5,13 +5,14 @@
 	<name>Sentry</name>
 	<summary>Sentry client</summary>
 	<description>A Sentry integration that sends unhandled exceptions to a Sentry instance to aggregate application crashes. You either have to set up your own Sentry instance or use your sentry.io account. See the admin documentation for how to configure this app.</description>
-	<version>8.12.3</version>
+	<version>8.12.3-dev.1</version>
```

Comment:
This repo uses release automation. Your *feat* commit will trigger a minor bump automatically. Please revert this line so I don't have to do that manually ;-)